src/coreclr/src/vm/jitinterface.cpp

```diff
@@ -444,7 +445,8 @@ CEEInfo::ConvToJitSig(
     CORINFO_SIG_INFO *    sigRet,
     MethodDesc *          pContextMD,
     bool                  localSig,
-    TypeHandle            contextType)
```

Comment:
Doc needs to be updated above. Is this a contract function? If not, can we clear it up the signature is a mess with a broad assortment of seemingly random arguments.

Reply:
This is not a contract function. I'll try cleaning up the signature.